DailyDemo.xcodeproj/project.pbxproj

```diff
@@ -7,15 +7,17 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		210630072B5EFA3800A6C8B4 /* LoopingVideoSource.swift in Sources */ = {isa = PBXBuildFile; fileRef = 210630062B5EFA3800A6C8B4 /* LoopingVideoSource.swift */; };
```

Comment:
should these new files also appear in this pr?

Reply:
Yep, that was the idea.
Also to keep it in sync with our private repo.

Reply:
@filipi87 is there a way to see the contents of LoopingVideoSource?

Reply:
My bad, I forgot to add them to git. 
Solving this issue in this [PR](https://github.com/daily-demos/daily-ios-demo/pull/27).